### ISSUE TITLE
Difference between paper and code

### ISSUE THREAD

**Issue by whikwon** [juntang-zhuang/LadderNet#4]
I've found 2 differences btw code and paper. If you check for me, it would be really helpful.

1. BatchNormalization is turned off in the code. https://github.com/juntang-zhuang/LadderNet/blob/35ca1703db8fa3ded185bee2425866fce01511d8/src/LadderNetv65.py#L25
2. I can't find shared-weights residual block. Every time you declare the network variable, new, independent residual block is created. isn't it?
https://github.com/juntang-zhuang/LadderNet/blob/35ca1703db8fa3ded185bee2425866fce01511d8/src/LadderNetv65.py#L92

**Comment by juntang-zhuang** [comment]
Thanks for your interest. 

For the first question, I forgot the detailed comparison when turning bn on or off. But you can easily turn it on for comparison. I might make a mistake in the paper with bn layer, since I directly copy the code from my local repo, it should be the same with the one generating results reported in my paper.

For the second question, shared-weights means sharing weights within a block, not all blocks are the same.  See the code here https://github.com/juntang-zhuang/LadderNet/blob/35ca1703db8fa3ded185bee2425866fce01511d8/src/LadderNetv65.py#L38 and https://github.com/juntang-zhuang/LadderNet/blob/35ca1703db8fa3ded185bee2425866fce01511d8/src/LadderNetv65.py#L44

BTW, the loss seems to be not defined in a clean way, since this is a toy repo when I just started deep learning and was not familiar with PyTorch. You can change the output of this network to a Conv layer without activation, and use CrossEntropyLoss or BCEWithLogitsLoss in PyTorch.

**Comment by whikwon** [comment]
Thanks for your reply. I understood what you meant. 

For the BN, I'll take an experiment with/without BN and share the result!
